### ISSUE TITLE
Margins image is not displaying in vignettes

### ISSUE THREAD

**Issue by dlcomeaux** [CMAP-REPOS/cmapplot#129]
For some reason, at least on my computer, the margins pngs are not displaying in the finalize vignette. Is anyone else having this issue?
![image](https://user-images.githubusercontent.com/58892984/114091333-2e98d000-987e-11eb-8500-2f6a3ef924f0.png)

**Comment by nmpeterson** [comment]
Most recent [pkgdown run](https://github.com/CMAP-REPOS/cmapplot/runs/2015464881?check_suite_focus=true)'s log says (under "Deploy package"):

> Copying 'man/figures/margins1.PNG' to 'reference/figures/margins1.PNG'
> Copying 'man/figures/margins2.PNG' to 'reference/figures/margins2.PNG'

Meanwhile, the vignette includes the following HTML to display the images:

```
Here, the margins are visualized as they impact a finalized plot that has a sidebar:

<img src="./../man/figures/margins1.png" width="100%" style="display: block; margin: auto;"/>

Here, the same margins are visualized as they impact a finalized plot with no sidebar:

<img src="./../man/figures/margins2.png" width="100%" style="display: block; margin: auto;"/>
```

I guess those file references should just be changed to say `./../reference` instead of `./../man`? Would this create any issues in terms of viewing vignettes locally?

**Comment by nmpeterson** [comment]
[This seems relevant.](https://pkgdown.r-lib.org/reference/build_articles.html#external-files)

**Comment by nmpeterson** [comment]
Fixed by #133.
